lstchain/reco/utils.py

```diff
@@ -203,16 +203,24 @@ def reco_source_position_sky(cog_x, cog_y, disp_dx, disp_dy, focal_length, point
     return camera_to_sky(src_x, src_y, focal_length, pointing_alt, pointing_az)
 
 
-def camera_to_sky(pos_x, pos_y, focal, pointing_alt, pointing_az):
+def camera_to_sky(pos_x, pos_y, focal, pointing_alt, pointing_az, time = None):
```

Comment:
maybe we should change the name of the function. As @MaxNoe pointed out in #420 , this method is converting to a `HorizonFrame`